src/license_manager_agent_ops.py

```diff
@@ -82,19 +111,16 @@ def install(self):
             "sacctmgr",
             "add",
             "user",
-            "license-manager",
-            f"Account=license-manager",
+            self._LICENSE_MANAGER_USER,
+            f"Account={self._LICENSE_MANAGER_ACCOUNT}",
             "AdminLevel=Operator",
             "-i",
         ]
         subprocess.call(add_to_account_cmd)
         logger.debug(f"license-manager-agent user added to account with operator admin level")
 
-        # Create log dir
-        if not self._LOG_DIR.exists():
-            self._LOG_DIR.mkdir(parents=True)
-        chown(self._LOG_DIR.as_posix(), self._SLURM_USER, self._SLURM_GROUP)
-
+    def install(self):
+        """Install license-manager-agent and setup ops."""
```

Comment:
It acts as a verb here, so I believe the correct is `set up`.
```suggestion
        """Install license-manager-agent and set up ops."""
```